item-spec/item-spec.md

```diff
@@ -167,6 +167,20 @@ or streamed. It is allowed to add additional fields.
 | href       | string | **REQUIRED.** Link to the asset object. Relative and absolute links are both allowed. |
 | title      | string | The displayed title for clients and users.                                            |
 | type       | string | [Media type](#media-types) of the asset.                                              |
+| role        | string | The semantic purpose of the asset.                                                   |
+
+**role** is intended to be a value that describes semantic usage of an asset, similarly to the use of `rel` in 
+Link. 
+ 
+#### Asset Role Types
+
+Like the Link `rel` field, the `role` field can be given any value.  These are a few standardized role names.
+
+| Role Name | Description                                                                           |
+| --------- | ------------------------------------------------------------------------------------- |
+| thumbnail |  An asset that represents a thumbnail of the item, typically a true color image (for items with assets in the visible wavelengths), lower-resolution (typically smaller 600x600 pixels), and typically a JPEG or PNG (suitable for display in a web browser). Multiple assets may have this purpose, but it recommended that the `type` and `role` be unique tuple. For example, Sentinel-2 L2A provides thumbnail images in both JPEG and JPEG2000 formats, and would be distinguished by their media types. |
+| overview  | An asset that represents a possibly larger view than the thumbnail of the Item , for example, a true color composite of multi-band data. |
+| metadata  | A metadata sidecar file describing the data in this item, for example the Landsat-8 MTL file. |
 
```

Comment:
Should we add an "analytic" role or something similar? What do we tag the full image files with?

Reply:
See #308 for related discussions (primary? analytic?)

Reply:
Thanks! I approved as is, so I'm happy to leave discussion there